packages/ui/src/components/Modal/modal.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { Modal as DefaultModal, ModalProps as DefaultModalProps } from 'antd'
+import classNames from 'classnames'
+import { Typography } from '../Typography'
+import { ModalFooter } from './footer'
+import { Cross } from './cross'
+
+type CondoModalProps = {
+    title: string
+    open: boolean
+    okText?: string
+    cancelText?: string
+}
+
+export type ModalProps = Pick<DefaultModalProps,
+'children'
+| 'width'
+| 'footer'
+| 'onOk'
+| 'onCancel'
+| 'confirmLoading'
+| 'className'
+| 'afterClose'
+| 'maskClosable'
+| 'destroyOnClose'
+| 'getContainer'
+| 'zIndex'
+> & CondoModalProps
+
+const MODAL_CLASS_PREFIX = 'condo-modal'
+
+const Modal: React.FC<ModalProps> = (props) => {
+    // NOTE The popup should have two widths, but so far only the size of the smallest width is known
+    // Therefore, for now, change the width to any. But by default we set 570
+    const { children, className, title, width = 570, footer, ...rest } = props
```

Comment:
There should be 3 sizes as far as I know...Small, big, and adaptive...Adaptive is used in billing for example